pages/profiles/prajwalajayaprakash.md

```diff
@@ -1,32 +1,44 @@
-# *Prajwala Jayaprakash*
-
-* Lives in New Paltz, New York
-* Eastern Time Zone
-* Using Windows 10
-
-### SUMMARY:
-> Electrical Engineering graduate with a focus on Computer Engineering from the State University of New York at New Paltz. Strong fundamentals in Digital Logic, VLSI design and Embedded Systems ( ARM Cortex ). Passionate about programming languages hence a self taught computer programmer.
-Recipient of OUTSTANDING GRADUATE award by the President of SUNY New Paltz.
-
-Linkedin: https://www.linkedin.com/in/prajwalajayaprakash/
-
-### SKILLS:
-> C | C++ | Java | Python | Arduino | Matlab | Cadence Virtuoso | MySQL
-
-### LANGUAGES:
-> English | Hindi | Kannada
-
-### PROJECTS:
-| Sl.No.   | Title   | Year   |
-|:---------:|-------|:------:|
-| 1. | AI Ping Pong Game | 2017 |
-| 2. | GetWeather - Internet of Things | 2017 |
-| 3. | 4 Bit adder subtractor in Cadence Virtuoso | 2016 |
-| 4. | Design of MIPS like processor | 2015 |
-| 5. | Modelling simulation and analysis of DC Motor | 2015 |
-| 6. | Wearable technology for women's safety | 2014 |
-| 7. | Automatic room light controller with visitor counter | 2014 |
-
-
-  
+prajwalajayaprakash-patch-1
+# *Prajwala Jayaprakash*               
+          
+ * Lives in New Paltz, New York
+ * Eastern Time Zone
+ * Using Windows 10
+ 
+ 
+ 
+ ### SUMMARY:
+ > Electrical Engineering graduate with a focus on Computer Engineering from the State University of New York at New Paltz. Strong fundamentals in Digital Logic, VLSI design and Embedded Systems ( ARM Cortex ). Passionate about programming languages hence a self taught computer programmer.
+ Recipient of OUTSTANDING GRADUATE award by the President of SUNY New Paltz.
+ 
+ Find me on Linkedin : [Linkedin Profile](https://www.linkedin.com/in/prajwalajayaprakash/)
+ 
+ ### SKILLS:
+ > C | C++ | Java | Python | Arduino | Matlab | Cadence Virtuoso | MySQL
+ 
+ ### LANGUAGES:
+ > English | Hindi | Kannada
+ 
+ ### PROJECTS:
+ | Sl.No.   | Title   | Year   |
+ |:---------:|-------|:------:|
+ | 1. | AI Ping Pong Game | 2017 |
+ | 2. | GetWeather - Internet of Things | 2017 |
+ | 3. | 4 Bit adder subtractor in Cadence Virtuoso | 2016 |
+ | 4. | Design of MIPS like processor | 2015 |
+ | 5. | Modelling simulation and analysis of DC Motor | 2015 |
+ | 6. | Wearable technology for women's safety | 2014 |
+ | 7. | Automatic room light controller with visitor counter | 2014 |
+
+Here are some pictures of my projects!
+
+
+![](http://imagizer.imageshack.us/v2/320x240q90/923/gIJX2m.jpg "AI Ping Pong Game")                         ![](http://imagizer.imageshack.us/v2/320x240q90/922/FGJE37.png "GetWeather - Internet of Things")
```

Comment:
Looks good so far.  Another picture should start next line.
And it would be better to name your image.